packages/theme-chalk/src/popper.scss

```diff
@@ -1,8 +1,9 @@
 @import 'mixins/mixins';
+@import 'mixins/var';
 @import 'common/var';
 
-:root {
-  --el-popper-border-radius: var(--el-popover-border-radius, 4px);
+@include b(popper) {
+  @include set-component-css-var('popper', $--popper);
 }
 
 @include b(popper) {
```

Comment:
why not include it here?

Reply:
Compiled as a class that contains variables and a class that does not contain variables.